src/components/Layout/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import SearchField from "../UI/SearchField";
+import { useDispatch } from "react-redux";
+import classes from "./Header.module.css";
```

Comment:
Try to use StyledComponents as everybody is using it and it will be good for you to get use to it. But, there is nothing wrong with this approach